pkg/security/probe/actions_linux_easyjson.go

```diff
@@ -1,5 +1,4 @@
 //go:build linux
-// +build linux
```

Comment:
Those are generated by easyjson, sadly we can't really clean them up. I'm surprised it's not making `security_go_generate_check` fail

Reply:
yeah as expected.. https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/1108570166